### ISSUE TITLE
Linux: App freezes after a scan

### ISSUE THREAD

**Issue by bydioeds** [rmcrackan/Libation#329]
After doing an scan the app freezes, to start the download of the book i need to close the app and re-open it.
Log after doing an scan and the app freezes:
```
 {}2022-07-28 15:13:45.507 +00:00 [DBG] (at AudibleUtilities.ApiExtended+<getItemsAsync>d__14.MoveNext()) Beginning library scan.
 {}2022-07-28 15:13:47.023 +00:00 [INF] (at AudibleApi.Api+<downloadItemPage>d__33.MoveNext()) Page 1: 2 results
 {}2022-07-28 15:13:47.027 +00:00 [DBG] (at AudibleUtilities.ApiExtended+<getItemsAsync>d__14.MoveNext()) Library scan complete. Found 2 books and series. Waiting on 0 series episode scans to complete.
 {}2022-07-28 15:13:47.028 +00:00 [DBG] (at AudibleUtilities.ApiExtended+<getItemsAsync>d__14.MoveNext()) Completed library scan.
 {}2022-07-28 15:13:47.109 +00:00 [INF] (at ApplicationServices.LibraryCommands+<ImportAccountAsync>d__15.MoveNext()) GetAllLibraryItems: Total count 2
 {}2022-07-28 15:13:47.114 +00:00 [INF] (at ApplicationServices.LibraryCommands+<ImportAccountAsync>d__15.MoveNext()) Begin scan for orphaned episode parents
 {}2022-07-28 15:13:47.134 +00:00 [INF] (at ApplicationServices.LibraryCommands+<ImportAccountAsync>d__15.MoveNext()) Orphan episode scan complete. New parents count -1
 {}2022-07-28 15:13:47.138 +00:00 [INF] (at ApplicationServices.LibraryCommands+<ImportAccountAsync>d__15.MoveNext()) Begin long-running import
 {}2022-07-28 15:13:47.391 +00:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts()) Book counts. {"total":2,"booksFullyBackedUp":0,"booksDownloadedOnly":0,"booksNoProgress":2,"booksError":0}
 {}2022-07-28 15:13:47.394 +00:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts()) PDF counts. {"total":0,"pdfsDownloaded":0,"pdfsNotDownloaded":0}
 {}2022-07-28 15:13:47.439 +00:00 [DBG] (at LibationSearchEngine.SearchEngine.Search(System.String)) original search string: {"searchString":null}
 {}2022-07-28 15:13:47.440 +00:00 [DBG] (at LibationSearchEngine.SearchEngine.Search(System.String)) formatted search string: {"searchString":"*:*"}
 {}2022-07-28 15:13:47.446 +00:00 [DBG] (at LibationSearchEngine.SearchEngine.generalSearch(System.String)) query: {"queryString":"*:*"}
 {}2022-07-28 15:13:47.449 +00:00 [DBG] (at LibationSearchEngine.SearchEngine.Search(System.String)) Hit(s): {"count":2}
 {}2022-07-28 15:13:47.528 +00:00 [INF] (at ApplicationServices.LibraryCommands+<ImportAccountAsync>d__15.MoveNext()) Import complete. New count 1
 {}
```
Log after re-opening and download the book:

```
{}2022-07-28 15:21:43.206 +00:00 [INF] (at AppScaffolding.LibationScaffolding.logStartupState(LibationFileManager.Configuration)) 


 {}2022-07-28 15:21:43.231 +00:00 [INF] (at AppScaffolding.LibationScaffolding.logStartupState(LibationFileManager.Configuration)) Begin. {"AppName":"Libation","Version":"8.3.0.1","Mode":"Release","LogLevel_Verbose_Enabled":true,"LogLevel_Debug_Enabled":true,"LogLevel_Information_Enabled":true,"LogLevel_Warning_Enabled":true,"LogLevel_Error_Enabled":true,"LogLevel_Fatal_Enabled":true,"BetaOptIn":false,"LibationFiles":"/home/pansuto/Libation","BooksDirectory":{"Path":"/home/pansuto/Libation/Books","ShortPathName":"/home/pansuto/Libation/Books","LongPathName":"/home/pansuto/Libation/Books","PathWithoutPrefix":"/home/pansuto/Libation/Books","$type":"LongPath"},"InProgress":"/tmp/Libation","DownloadsInProgressDirectory":{"Path":"/tmp/Libation/DownloadsInProgress","ShortPathName":"/tmp/Libation/DownloadsInProgress","LongPathName":"/tmp/Libation/DownloadsInProgress","PathWithoutPrefix":"/tmp/Libation/DownloadsInProgress","$type":"LongPath"},"DownloadsInProgressFiles":0,"DecryptInProgressDirectory":{"Path":"/tmp/Libation/DecryptInProgress","ShortPathName":"/tmp/Libation/DecryptInProgress","LongPathName":"/tmp/Libation/DecryptInProgress","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress","$type":"LongPath"},"DecryptInProgressFiles":0}
 {}2022-07-28 15:21:44.766 +00:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts()) Book counts. {"total":2,"booksFullyBackedUp":0,"booksDownloadedOnly":0,"booksNoProgress":2,"booksError":0}
 {}2022-07-28 15:21:44.860 +00:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts()) PDF counts. {"total":0,"pdfsDownloaded":0,"pdfsNotDownloaded":0}
 {}2022-07-28 15:21:50.402 +00:00 [INF] (at LibationAvalonia.Views.MainWindow.ProductsDisplay_LiberateClicked(System.Object, DataLayer.LibraryBook)) Begin single book backup of 7/28/2022 [B0B5GMHL7B] Ucrania: El camino hacia la guerra
 {}2022-07-28 15:21:50.463 +00:00 [INF] (at LibationAvalonia.ViewModels.ProcessQueueViewModel.AddDownloadDecrypt(System.Collections.Generic.IEnumerable`1[[DataLayer.LibraryBook, DataLayer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]])) Queueing 1 books
 {}2022-07-28 15:21:50.511 +00:00 [INF] (at LibationAvalonia.ViewModels.ProcessQueueViewModel+<QueueLoop>d__60.MoveNext()) Begin processing queue
 {}2022-07-28 15:21:50.541 +00:00 [INF] (at LibationAvalonia.ViewModels.ProcessQueueViewModel+<QueueLoop>d__60.MoveNext()) Begin processing queued item. 7/28/2022 [B0B5GMHL7B] Ucrania: El camino hacia la guerra
 {}2022-07-28 15:21:50.561 +00:00 [INF] (at FileLiberator.Processable+<ProcessSingleAsync>d__23.MoveNext()) Begin ProcessSingleAsync {"Title":"Ucrania: El camino hacia la guerra","AudibleProductId":"B0B5GMHL7B","Locale":"spain","Account":"[...].i[...]2@g[...]l.c[...]m"}
 {}2022-07-28 15:21:50.583 +00:00 [DBG] (at FileLiberator.Processable.OnBegin(DataLayer.LibraryBook)) Event fired {"Name":"Begin","Book":["B0B5GMHL7B","Ucrania: El camino hacia la guerra","spain","[...].i[...]2@g[...]l.c[...]m"]}
 {}2022-07-28 15:21:50.596 +00:00 [INF] (at LibationAvalonia.LogMe+<>c.<.ctor>b__9_0(System.Object, System.String)) Automated backup: 
Download & Decrypt Step, Begin: [B0B5GMHL7B] Ucrania: El camino hacia la guerra
 {}2022-07-28 15:21:50.607 +00:00 [INF] (at AudibleUtilities.ApiExtended+<CreateAsync>d__9.MoveNext()) {"Username":"[...].i[...]2@g[...]l.c[...]m","LocaleName":"spain"}
 {}2022-07-28 15:21:52.207 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) No file to delete: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"}}
 {}2022-07-28 15:21:52.255 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Begin download and convert Aaxc To "M4b"
 {}2022-07-28 15:21:52.262 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Begin Step 1: Get Aaxc Metadata
 {}2022-07-28 15:21:52.909 +00:00 [DBG] (at FileLiberator.AudioDecodable.OnTitleDiscovered(System.Object, System.String)) Event fired {"Name":"TitleDiscovered","Title":"Ucrania: El camino hacia la guerra"}
 {}2022-07-28 15:21:52.911 +00:00 [DBG] (at FileLiberator.AudioDecodable.OnAuthorsDiscovered(System.Object, System.String)) Event fired {"Name":"AuthorsDiscovered","Authors":"Alejandro López Canorea"}
 {}2022-07-28 15:21:52.912 +00:00 [DBG] (at FileLiberator.AudioDecodable.OnNarratorsDiscovered(System.Object, System.String)) Event fired {"Name":"NarratorsDiscovered","Narrators":"Alejandro López Canorea"}
 {}2022-07-28 15:21:52.913 +00:00 [DBG] (at FileLiberator.AudioDecodable.OnCoverImageDiscovered(System.Byte[])) Event fired {"Name":"CoverImageDiscovered","CoverImageBytes":60827}
 {}2022-07-28 15:21:52.916 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Completed Step 1: Get Aaxc Metadata
 {}2022-07-28 15:21:52.917 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Begin Step 2: Download Decrypted Audiobook
 {}2022-07-28 15:21:52.919 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) No file to delete: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"}}
 {}2022-07-28 15:21:52.985 +00:00 [INF] (at FileLiberator.Processable.OnFileCreated(DataLayer.LibraryBook, System.String)) File created {"Name":"FileCreated","AudibleProductId":"B0B5GMHL7B","path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b"}
 {}2022-07-28 15:22:12.467 +00:00 [INF] (at FileLiberator.Processable.OnFileCreated(DataLayer.LibraryBook, System.String)) File created {"Name":"FileCreated","AudibleProductId":"B0B5GMHL7B","path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b"}
 {}2022-07-28 15:22:12.471 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Completed Step 2: Download Decrypted Audiobook
 {}2022-07-28 15:22:12.476 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Begin Step 3: Create Cue
 {}2022-07-28 15:22:12.491 +00:00 [INF] (at FileLiberator.Processable.OnFileCreated(DataLayer.LibraryBook, System.String)) File created {"Name":"FileCreated","AudibleProductId":"B0B5GMHL7B","path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue"}
 {}2022-07-28 15:22:12.495 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Completed Step 3: Create Cue
 {}2022-07-28 15:22:12.500 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Begin Step 4: Cleanup
 {}2022-07-28 15:22:12.508 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) Attempt to delete file: {"source":{"Path":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","ShortPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","LongPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","PathWithoutPrefix":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","$type":"LongPath"}}
 {}2022-07-28 15:22:12.510 +00:00 [INF] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) File successfully deleted: {"source":{"Path":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","ShortPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","LongPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","PathWithoutPrefix":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.json","$type":"LongPath"}}
 {}2022-07-28 15:22:12.511 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) Attempt to delete file: {"source":{"Path":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","ShortPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","LongPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","PathWithoutPrefix":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","$type":"LongPath"}}
 {}2022-07-28 15:22:12.618 +00:00 [INF] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) File successfully deleted: {"source":{"Path":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","ShortPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","LongPathName":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","PathWithoutPrefix":"/tmp/Libation/DownloadsInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.aaxc","$type":"LongPath"}}
 {}2022-07-28 15:22:12.629 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Completed Step 4: Cleanup
 {}2022-07-28 15:22:12.631 +00:00 [INF] (at AaxDecrypter.AaxcDownloadSingleConverter+<RunAsync>d__1.MoveNext()) Completed download and convert Aaxc To "M4b"
 {}2022-07-28 15:22:12.663 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) No file to delete: {"source":{"Path":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"}}
 {}2022-07-28 15:22:12.665 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) Attempt to create directory: {"dir":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]"}
 {}2022-07-28 15:22:12.666 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) Attempt to move file: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"},"destination":{"Path":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"}}
 {}2022-07-28 15:22:12.679 +00:00 [INF] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) File successfully moved: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"},"destination":{"Path":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"}}
 {}2022-07-28 15:22:12.681 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) No file to move: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.m4b","$type":"LongPath"}}
 {}2022-07-28 15:22:12.698 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass23_0.<SaferDelete>b__0()) No file to delete: {"source":{"Path":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","ShortPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","LongPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","PathWithoutPrefix":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","$type":"LongPath"}}
 {}2022-07-28 15:22:12.705 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) Attempt to create directory: {"dir":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]"}
 {}2022-07-28 15:22:12.706 +00:00 [DBG] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) Attempt to move file: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","$type":"LongPath"},"destination":{"Path":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","ShortPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","LongPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","PathWithoutPrefix":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","$type":"LongPath"}}
 {}2022-07-28 15:22:12.708 +00:00 [INF] (at FileManager.FileUtility+<>c__DisplayClass24_0.<SaferMove>b__0()) File successfully moved: {"source":{"Path":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","ShortPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","LongPathName":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","PathWithoutPrefix":"/tmp/Libation/DecryptInProgress/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","$type":"LongPath"},"destination":{"Path":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","ShortPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","LongPathName":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","PathWithoutPrefix":"/home/pansuto/Libation/Books/Alejandro López Canorea - Ucrania - [Alejandro López Canorea][B0B5GMHL7B]/Alejandro López Canorea - Ucrania꞉ El camino hacia la guerra.cue","$type":"LongPath"}}
 {}2022-07-28 15:22:13.315 +00:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts()) Book counts. {"total":2,"booksFullyBackedUp":1,"booksDownloadedOnly":0,"booksNoProgress":1,"booksError":0}
 {}2022-07-28 15:22:13.316 +00:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts()) PDF counts. {"total":0,"pdfsDownloaded":0,"pdfsNotDownloaded":0}
 {}2022-07-28 15:22:13.318 +00:00 [DBG] (at FileLiberator.Processable.OnCompleted(DataLayer.LibraryBook)) Event fired {"Name":"Completed","Book":["B0B5GMHL7B","Ucrania: El camino hacia la guerra","spain","[...].i[...]2@g[...]l.c[...]m"]}
 {}2022-07-28 15:22:13.320 +00:00 [INF] (at LibationAvalonia.LogMe+<>c.<.ctor>b__9_0(System.Object, System.String)) Automated backup: Download & Decrypt Step, Completed: [B0B5GMHL7B] Ucrania: El camino hacia la guerra
 {}2022-07-28 15:22:13.326 +00:00 [INF] (at LibationAvalonia.ViewModels.ProcessQueueViewModel+<QueueLoop>d__60.MoveNext()) Completed processing queued item: 7/28/2022 [B0B5GMHL7B] Ucrania: El camino hacia la guerra
Result: "Success"
 {}2022-07-28 15:22:13.334 +00:00 [INF] (at LibationAvalonia.ViewModels.ProcessQueueViewModel+<QueueLoop>d__60.MoveNext()) Completed processing queue
 {}                               
```

The book is downloaded and decrypted correctly, the scan adds all the books correctly,  the only issue is that weird freeze that happens after scanning.

**Comment by bydioeds** [comment]
> Which distribution and version are you running?

ubuntu 20.04 with icewm window manager if that matters.

**Comment by rmcrackan** [comment]
Fixed in [v8.3.1](https://github.com/rmcrackan/Libation/releases/latest). Thank you for your patience.

**Comment by Mbucari** [comment]
This experience has made me realize how important is to have an auto-update function in the Linux release (I know, I'm late to the party).  I'm working on that next.
